modules/tinymce/src/plugins/codesample/test/ts/webdriver/CodeSampleCopyAndPasteTest.ts

```diff
@@ -48,7 +48,8 @@ describe('webdriver.tinymce.plugins.codesample.CodeSampleCopyAndPasteTest', () =
     hook.editor().setContent('');
   });
 
-  (browser.isSafari() ? it.skip : it)('TINY-8861: press enter after pasting a code sample should not add a newline inside the code', async () => {
+  // Chromium requires clipboard permissions to be granted, so we can't run the test (and there is no bug)
+  (browser.isChromium() || browser.isSafari() ? it.skip : it)('TINY-8861: press enter after pasting a code sample should not add a newline inside the code', async () => {
```

Comment:
Isn't there a webdriver setting we can use to bypass this? Surely other developers who need the permission API have tests...

Reply:
the permission for `clipboard-write` was removed as a valid entry quite a while ago... I'm still trying to return the correct state with `navigator.permissions.query`, it seems it is always returning `true` even when I denied the prompt 
https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1245

Reply:
As discussed in Slack, the answer is "yes but bedrock needs to be updated"